apis/v1beta1/types.go

```diff
@@ -32,7 +32,7 @@ type ProviderConfigStatus struct {
 
 // +kubebuilder:object:root=true
 
-// A ProviderConfig configures a Template JET provider.
+// A ProviderConfig configures a Template provider.
```

Comment:
Should we also change the category below from `templatejet` to `template` ?

Reply:
Thanks for catching!